org.civicrm.reqangular/src/common/services/api.js

```diff
@@ -121,9 +122,13 @@ define([
             action: action
           }
         }).then(function (response) {
-
-          return response.data;
+          if(!!response.data.is_error) {
```

Comment:
the whole `then` function is the same in both `sendGET` and `sendPOST`, meaning you can put it into a separate function and just pass that function to then
```js
function commonHandler(response) {
  // ..
}

// ...

$http.get({
  // ...
})
.then(commonHandler);